Assets/AWSIM/Scripts/Vehicles/Vehicle.cs

```diff
@@ -424,8 +426,31 @@ void UpdateWheelsForce(float acceleration)
                 var perWheelAcceleration = acceleration / wheels.Length;
 
                 // Update the acceleration output by each wheel.
-                foreach (var wheel in wheels)
-                    wheel.UpdateWheelForce(perWheelAcceleration);
+                var dif1 = Mathf.Sin(Mathf.Asin(1 / Mathf.Sqrt(Mathf.Pow((1 / Mathf.Tan(Mathf.Abs(SteerAngle) * Mathf.Deg2Rad) + tread / wheelBase), 2) + 1))) / Mathf.Sin(SteerAngle * Mathf.Deg2Rad);
```

Comment:
Is it possible to leave comments on the calculations and logic of dif1, dif2, dif3, etc. so that other developers can easily understand them?

Reply:
Added some comments of dif1, dif2, dif3.